server/routes/pool-management/deferral-maintenance/deferral-maintenance.controller.js

```diff
@@ -178,15 +185,26 @@
         });
       }
 
-      data.queryTotal = data.deferrals.length;
+      if (sortBy) {
+        if (sortBy !== 'jurorNumber' || sortBy !== 'poolNumber') {
+          data.deferrals.sort((a, b) => {
+            const sortA = a[sortBy].toUpperCase();
+            const sortB = b[sortBy].toUpperCase();
+            if (sortA < sortB) {
+              return sortOrder === 'ascending' ? -1 : 1;
+            }
+            if (sortA > sortB) {
+              return sortOrder === 'ascending' ? 1 : -1;
+            }
 
-      if (currentPage > 1) {
-        offset = modUtils.constants.PAGE_SIZE * (currentPage - 1);
+            return 0;
+          });
```

Comment:
Could we use a `localeCompare()` on this sort function?
could probably write something like

```js
data.deferrals.sort((a, b) => {
  if (ascending) a.toUpperCase().localCompare(b.toUpperCase())
  else .... the opposite
})
```

Reply:
Updated to use localeCompare() :)